website/docs/r/nsxt_alb_virtual_service.html.markdown

```diff
@@ -129,6 +129,41 @@ resource "vcd_nsxt_alb_virtual_service" "test" {
 }
 ```
 
+## Example Usage (VCD 10.4.1+ - Virtual Service Transparent mode and Pool Group Membership)
+```hcl
+data "vcd_nsxt_ip_set" "frontend" {
+  org             = "my-org" # Optional
+  edge_gateway_id = data.vcd_nsxt_edgegateway.main.id
+  name            = "frontend-servers"
+}
+
+resource "vcd_nsxt_alb_pool" "test" {
+  org             = "my-org"
+  name            = "test-pool"
+  edge_gateway_id = data.vcd_nsxt_edgegateway.existing.id
+  member_group_id = data.vcd_nsxt_ip_set.frontend.id
+}
+
+resource "vcd_nsxt_alb_virtual_service" "test" {
+  org = "my-org"
+
+  name            = "new-virtual-service"
+  edge_gateway_id = data.vcd_nsxt_edgegateway.existing.id
+
+  # Preserve Client IP - can only be enabled in VCD 10.4.1+ (must also be enabled in `vcd_nsxt_alb_settings`)
```

Comment:
BTW, what happens if user enables this setting on lower VCD version?

Reply:
We have validation for it https://github.com/vmware/terraform-provider-vcd/pull/1024/files#diff-818eadb1083a77a7d7e5f532244049aa39ef464d5fd645518386c9d48e85392eR267-R269

Without it, one would get an API error of an unrecognized field on older versions.